source/collection-management/timeseries-collection.yml

```diff
@@ -0,0 +1,63 @@
+description: "timeseries-collection"
+
+schemaVersion: "1.0"
+
+runOnRequirements:
+  - minServerVersion: "4.9"
```

Comment:
Something else that occurred to me. Even if this functionality is in the v4.9 branch, since we already have 5.0 releases I think it makes sense to just use 5.0 here.